src/DonationForms/resources/app/DonationFormApp.tsx

```diff
@@ -59,8 +59,10 @@ function App({form}: { form: DonationForm }) {
         return (
             <DonationFormSettingsProvider value={{...form.settings,...donationFormNodeSettings}}>
                 <DonationFormStateProvider initialState={initialState}>
-                    {!form.design?.includeHeaderInMultiStep && form.settings.showHeader && <Header form={form} />}
-                    <MultiStepForm form={form} />
+                    <div className="givewp-donation-form__container">
```

Comment:
There is already a div that wraps the header and form with a classname "givewp-donation-form" - is there any compelling reason to add this as well or could you achieve the styling with the existing div?

Reply:
I needed that for responsive as with the detached header, I had to created a attached-like style for the form and I couldn't achieve that using the form div because I also needed an extra div for the form padding.

Reply:
Okay, my main hesitation is (although it doesn't seem like much) it's _technically_ a breaking change for styles because a customer could be using custom css and targeting the current structure.  I think it would be worth a little exploring if we can achieve with css grid - otherwise this would be an update I would want to make to default form as well so the base structure is the same across forms layouts.

Reply:
I can remove that container then rely on the media queries to manage the margins. I'll let you know when that's ready for review.

Reply:
@jonwaldstein refactored that on 582746161c232c167310ed781f91573a01188088

Reply:
Great, i think that is a safer approach for now, thanks!